src/core/tests/CMakeLists.txt

```diff
@@ -38,9 +38,9 @@ ov_add_test_target(
         LINK_LIBRARIES
             common_test_utils
             openvino::reference
-            openvino::builders
             openvino::util
             openvino::shape_inference
+            ngraph_obj
```

Comment:
looks like it should not be required

Reply:
It is. Tests use these functions. I moved the functions to core part. So now core tests requires static linkage to core lib.